wp-content/themes/sceraTheatre/scss/base.scss

```diff
@@ -64,3 +76,21 @@ img {
 		color: $color-blue-teal;
 	}
 }
+
+.text-section {
+	font-size: 20px;
+	color: $color-grey-dark;
+	font-family: $font-oxygen;
+}
+
+.title-section {
+	padding: 3.8rem 2rem 4.8rem 2rem;
```

Comment:
Just as an FYI, the last `2rem` is redundant here. It can be `3.8rem 2rem 4.8rem` and it will assume 2rem for the left padding.